platform-tests/src/test/java/org/junit/platform/suite/engine/SuiteEngineTests.java

```diff
@@ -74,6 +76,21 @@ void selectClasses() {
 		// @formatter:on
 	}
 
+	@Test
+	void selectMethods() {
+		// @formatter:off
+		EngineTestKit.engine(ENGINE_ID)
+				.selectors(selectClass(SelectMethodsSuite.class))
+				.execute()
+				.testEvents()
+				.assertThatEvents()
+				.haveExactly(1, event(test(SelectMethodsSuite.class.getName()), finishedSuccessfully()));
+				//.doNotHave(event(test(MultipleTestsTestCase.class.getDeclaredMethod("test2", new Class[0]).getName()), finishedSuccessfully()));
```

Comment:
You can use ` event(test(MultipleTestsTestCase.class.getName(), "test()")` to target a combination of unique Id part and method name. The `()` are important to getting the method name to work.